### PR TITLE
Get weets

### PR DESCRIPTION
**Pull request by ricardoamoreno**
## Summary

- get /weets endpoint added
- pagination added
- getWeets tests added

## Known Issues

None

## Trello Card

[Obetner Weets](https://trello.com/c/jcnaEOOx)

### CODE REVIEW

File: test/get_weets.spec.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    .query({ page, limit });
+
+let userToken = null;
+beforeAll(async () => {
+  await createUser(mockUser);
+  const signInData = await signIn({ email: mockUser.email, password: mockUser.password });
+  userToken = signInData.body.token;
+});
+
+describe('GET /users/sessions', () => {
```
Comment (wgermancutraro) [inline test/get_weets.spec.js]: The URL is probably wrong

Comment (wgermancutraro) [inline test/get_weets.spec.js]: If the following that I wrote is in another card, discard the following: I would add more tests on pagination

**Review by wgermancutraro — CHANGES_REQUESTED** [review]

File: test/get_weets.spec.js (hunk 1 above)
Comment (ricardoamoreno) [reply test/get_weets.spec.js]: yes. it's wrong

Comment (ricardoamoreno) [reply test/get_weets.spec.js]: i implemented factory and test for pagination

File: test/get_weets.spec.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+factory.define('user', User, {});
+factory.define('weet', Weet, { content: 'loremIpsum', userId: factory.assoc('user', 'id') });
+
+const createUser = user => request.post('/users').send(user);
+const signIn = user => request.post('/users/sessions').send(user);
+
+let userToken = null;
+beforeAll(async done => {
+  await createUser(mockUser);
+  const signInData = await signIn({ email: mockUser.email, password: mockUser.password });
```
Comment (wgermancutraro) [inline test/get_weets.spec.js]: You can mock the token verify

**Review by wgermancutraro — CHANGES_REQUESTED** [review]

File: test/get_weets.spec.js (hunk 2 above)
Comment (ricardoamoreno) [reply test/get_weets.spec.js]: done

**Review by wgermancutraro — APPROVED** [review]
🎉

**Pull request merged**